org.aposin.licensescout.core/launch/org.aposin.licensescout.core_license_format.launch

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">
-    <booleanAttribute key="M2_DEBUG_OUTPUT" value="false"/>
```

Comment:
This indentation changes hide the real modification - could you revert where there is no change in the line?

Reply:
I found out that the indentation differences came from using differen Eclipse versions. At home I was using Eclipse Oxygen. At work we are using Eclipse 4.9, which does the formatting without indentations. Since this is the newer one, it makes sense to change to formatting once. Otherwise, if we keep the formatting of older Eclipse versions, we will see unmotivated formatting changes again and again. This also applies to the other launch files.